setup.py

```diff
@@ -14,14 +29,15 @@
     description="Text Similarity Index Processor",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="https://github.com/bkk003/text_de_duplication_monitoring",
+    url="https://github.com/philips-software/TextSimilarityProcessor",
     # packages=setuptools.find_packages(exclude=['test', '*.test', '*.test.*']),
-    packages=setuptools.find_packages(include=['similarity_processor'], exclude=['test', '*.test', '*.test.*']),
+    packages=setuptools.find_packages(include=['similarity_processor', 'spell_check'],
+                                      exclude=['test', '*.test', '*.test.*']),
     classifiers=[
         "Programming Language :: Python :: 3",
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
     ],
     install_requires=required,
-    python_requires='>=3.6',
-)
\ No newline at end of file
+    python_requires='>=3.7',
+)
```

Comment:
A new line here may also be helpful, to comply with standards: https://unix.stackexchange.com/a/18789